core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -331,27 +327,34 @@ angular.module('oppia').directive('audioTranslationBar', [
             });
           };
           var toggleStartAndStopRecording = function() {
-            if ($scope.isAudioAvailable) {
```

Comment:
Why has this check been removed?

Reply:
`toggleStartAndStopRecording` is only called when `isAudioAvailable` is `false`, as you can see [here](https://github.com/oppia/oppia/blob/e6f903f4fa7a644be5936dd0f0d3c21ca03ad333/core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts#L528-L531). So this `if` will never be evaluated.

Reply:
Ah, I see. Thanks for explaining!